### PR TITLE
added vue datepicker due to need for choosing a date

### PR DESCRIPTION
**Pull request by p3t3r67x0**

### CODE REVIEW

File: frontend/app/pages/results.vue [hunk 1]
```diff
@@ -82,19 +90,37 @@ export default {
...
+    notAfterToday(date) {
+      return date > this.today
+    },
+    async getFilterResult() {
+      const dateParam = `${this.date !== null ? `&date=${this.date}`: ''}`
+      const countyParam = `${this.county !== null ? `&county=${this.county}`: ''}`
+      const zipCodeParam = `${this.zipCodeRegex.test(this.zipCode) ? `&zip_code=${this.zipCode}`: ''}`
+
+      try {
+        const results = await this.$axios.$get(`${process.env.API_URL}/report/result/?new_format${dateParam}${zipCodeParam}${countyParam}`)
```
Comment (40818419) [inline frontend/app/pages/results.vue]: lets use query-string module https://www.npmjs.com/package/query-string to generate filter params, so we could have build url out of 

```
const queryParams = {
   date: ...,
   county: ...,
   zipCode: ...
}

queryString.stringify(queryParams)
```

File: frontend/app/pages/results.vue [hunk 2]
```diff
@@ -4,7 +4,12 @@
     <h1 class="text-2xl mb-3 text-blue-600">
       Übersicht der eingereichten Daten
     </h1>
-    Filter: <input type="text" class="w-1/4 border border-blue-600 rounded text-xl p-2 mb-3" v-model="zipCode" placeholder="PLZ">
+    <h3 class="text-xl">Filter</h3>
+    <input type="text" class="sm:w-1/4 border border-blue-600 rounded text-xl p-2 mb-3" v-model="county" placeholder="Landkreis">
+    <input type="text" class="sm:w-1/4 border border-blue-600 rounded text-xl p-2 mb-3" v-model="zipCode" placeholder="PLZ">
+    <client-only>
+      <date-picker v-model="date" valueType="format" :disabled-date="notAfterToday" class="sm:w-1/4 border border-blue-600 rounded text-xl mb-3" placeholder="YYYY-MM-DD"></date-picker>
```
Comment (40818419) [inline frontend/app/pages/results.vue]: lets use german format "DD-MM-YYYY" ?

Comment (p3t3r67x0) [reply frontend/app/pages/results.vue]: Die API von Bernhard erwartet leider das englishe Foramt

File: frontend/app/pages/results.vue (hunk 1 above)
Comment (p3t3r67x0) [reply frontend/app/pages/results.vue]: ok, gute idee

File: frontend/app/pages/results.vue (hunk 2 above)
Comment (40818419) [reply frontend/app/pages/results.vue]: ja, aber dann sollen wir label von values unterscheiden

Comment (p3t3r67x0) [reply frontend/app/pages/results.vue]: done

File: frontend/app/pages/results.vue (hunk 1 above)
Comment (p3t3r67x0) [reply frontend/app/pages/results.vue]: done

**Pull request merged**